python/lsst/afw/display/ds9.py

```diff
@@ -40,19 +40,14 @@
     except Exception as e:
         # No usable version of display_ds9.
         # Let's define a version of getDisplay() which will throw an exception.
-        e.args = ["%s (is display_ds9 setup?)" % e]
 
-        # TODO: once we abandon Python 2 use the following simpler code:
-        # def getDisplay(*args, exception=e, **kwargs):
-        #     raise exception
-        class _RaiseException(object):
-            def __init__(self, exception):
-                self.exception = exception
```

Comment:
@rowen, do you mean you would prefer to change this line to something like:
```python
self.exception = type(e)("%s (is display_d9 setup?)" % e)
```
(and remove the `e.args` bit from above)?